samples/excel/89-preview-apis/pivot-table-filters-and-summaries.yaml

```diff
@@ -0,0 +1,213 @@
+            });
+        }
+
+        async function setup() {
+            await Excel.run(async (context) => {
+                const dataSheet = await OfficeHelpers.ExcelUtilities
+                    .forceCreateSheet(context.workbook, "Data");
+                const pivotSheet = await OfficeHelpers.ExcelUtilities
+                    .forceCreateSheet(context.workbook, "Pivot");
+
+                const data = [["Farm", "Type", "Classification", "Crates Sold at Farm", "Crates Sold Wholesale"],
+                ["A Farms", "Lime", "Organic", 300, 2000],
+                ["A Farms", "Lemon", "Organic", 250, 1800],
+                ["A Farms", "Orange", "Organic", 200, 2200],
+                ["B Farms", "Lime", "Conventional", 80, 1000],
+                ["B Farms", "Lemon", "Conventional", 75, 1230],
+                ["B Farms", "Orange", "Conventional", 25, 800],
+                ["B Farms", "Orange", "Organic", 20, 500],
+                ["B Farms", "Lemon", "Organic", 10, 770],
+                ["B Farms", "Kiwi", "Conventional", 30, 300],
+                ["B Farms", "Lime", "Organic", 50, 400],
+                ["C Farms", "Apple", "Organic", 275, 220],
+                ["C Farms", "Kiwi", "Organic", 200, 120],
+                ["D Farms", "Apple", "Conventional", 100, 3000],
+                ["D Farms", "Apple", "Organic", 80, 2800],
+                ["E Farms", "Lime", "Conventional", 160, 2700],
+                ["E Farms", "Orange", "Conventional", 180, 2000],
+                ["E Farms", "Apple", "Conventional", 245, 2200],
+                ["E Farms", "Kiwi", "Conventional", 200, 1500],
+                ["F Farms", "Kiwi", "Organic", 100, 150],
+                ["F Farms", "Lemon", "Conventional", 150, 270]];
+
+                const range = dataSheet.getRange("A1:E21");
+                range.values = data;
+                range.format.autofitColumns();
+                pivotSheet.activate();
+
+                context.workbook.worksheets.getActiveWorksheet()
+                    .pivotTables.add("Farm Sales", "Data!A1:E21", "Pivot!A2");
+                let pivotTable = context.workbook.worksheets.getActiveWorksheet().pivotTables.getItem("Farm Sales");
+                pivotTable.rowHierarchies.add(pivotTable.hierarchies.getItem("Farm"));
+                pivotTable.rowHierarchies.add(pivotTable.hierarchies.getItem("Type"));      
+                pivotTable.dataHierarchies.add(pivotTable.hierarchies.getItem("Crates Sold at Farm"));
+                pivotTable.dataHierarchies.add(pivotTable.hierarchies.getItem("Crates Sold Wholesale"));
+
+                await context.sync();
+            });
+        }
+
+        async function deletePivot() {
+            await Excel.run(async (context) => {
+                context.workbook.worksheets.getItem("Pivot").pivotTables.getItem("Farm Sales").delete();
+                context.workbook.worksheets.getActiveWorksheet().getRange("B27:C27").delete(Excel.DeleteShiftDirection.up);
```

Comment:
Please add a comment explaining the need for this line:
`context.workbook.worksheets.getActiveWorksheet().getRange("B27:C27").delete(Excel.DeleteShiftDirection.up); `